keel-veto/src/test/kotlin/com/netflix/spinnaker/keel/veto/DummyVeto.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.veto
+
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.api.ResourceId
+import com.netflix.spinnaker.keel.api.application
+import com.netflix.spinnaker.keel.api.id
+
```

Comment:
Created a dummy veto to test the veto enforcer logic with since before it was using the application veto.

Reply:
Does the concept of vetoing even apply anymore? Did we foresee any additional use cases beyond the application veto?

Reply:
We have the unhappy veto as well. We do foresee other use cases - like region evac, etc